src/composables/transactionAndTokenFilter.ts

```diff
@@ -26,10 +26,11 @@ export const useTransactionAndTokenFilter = () => {
   );
 
   const filtersConfig = ref<IFilters<ObjectValues<typeof FILTER_MODE>>>({
-    all: { name: i18n.t('common.all') },
-    in: { name: i18n.t('filters.in') },
-    out: { name: i18n.t('filters.out') },
-    dex: { name: i18n.t('filters.dex') },
+    // @ts-ignore
```

Comment:
Why this is added here? Maybe there's better way of satisfying the TS?

Reply:
The type coming from VueI18n library is considered by TS _excessively deep and possibly infinite_.
So it throws this error: `Type instantiation is excessively deep and possibly infinite.`
I believe we can `@ts-ignore` it. 
Perhaps a better way would be to use `@ts-expect-error` with an explanation line `type coming from VueI18n is excessively deep and possibly infinite`

Reply:
OK, that's really weird. So I agree that this requires some additional comment and maybe we would be able to remove the ignore in the future.